tests/test_alias.rs

```diff
@@ -101,7 +101,7 @@ fn test_alias_global_args_before_and_after() {
     let repo_path = test_env.env_root().join("repo");
     test_env.add_config(
         br#"[alias]
-    l = ["log", "-T", "commit_id"]
+    l = ["log", "-T", "commit_id", "-r", "remote_branches().. | (remote_branches()..)-"]
```

Comment:
nit: It looks like these tests don't have any remote branches, so using the `all()` revset should work. I think that's clearer because it doesn't make the reader wonder where this revset comes from.

Reply:
It looks like we need the same change in `test_workspaces_forget()`. It's a little unfortunate that we break plain `jj log` in these cases, but they're rare enough that it's probably okay.

Maybe it's better to make `@` resolve the empty set than to an error, as you suggested, but I'm not sure. That might also surprise people who expected `@` would resolve to a commit.

Reply:
> Maybe it's better to make @ resolve the empty set than to an error, as you suggested, but I'm not sure

Yeah, silently doing something nonobvious isn't great either. I'm happy to leave that for future work, if ever.

> using the all() revset should work

Good catch, that's much nicer.